cmd/kubent/main.go

```diff
@@ -154,10 +154,11 @@ func main() {
 		log.Fatal().Err(err).Str("name", "Rego").Msg("Failed to filter results")
 	}
 
-	printer, err := printer.NewPrinter(config.Output)
+	printer, err := printer.NewPrinter(config.Output, config.OutputFile)
```

Comment:
shouldnt this be only passed if the flag is set?

Reply:
I would say no, given go doesn't really do optional args, we just pass the default (which is stdout `-`), same as we do just a couple of lines above, e.g. `loadedRules, err := rules.FetchRegoRules(additionalKinds)`.